subsys/bluetooth/host/att.c

```diff
@@ -86,8 +86,8 @@ typedef void (*bt_att_tx_cb_t)(struct bt_conn *conn,
 			       struct bt_att_tx_meta_data *user_data);
 
 struct bt_att_tx_meta_data {
-	sys_snode_t tx_cb_queue_node;
-	bt_att_tx_cb_t chan_cb;
+	uint8_t opcode;
+	int err;
 	struct bt_att_chan *att_chan;
 	uint16_t attr_count;
```

Comment:
If we move `attr_count` next to `opcode`, don't we save a word in memory?

Reply:
something like that, yeah. don't resolve the conversation, I'll add another commit for re-ordering.